atomicapp/applogging.py

```diff
@@ -154,7 +158,10 @@ def setup_logging(verbose=None, quiet=None, logtype=None):
 
             # configure logger for basic no color printing to stdout
             handler = logging.StreamHandler(stream=sys.stdout)
-            formatter = customOutputFormatter(formatstr)
+
+            # We include %s in order to specify Unix formatted time output
+            formatter = customOutputFormatter(*formattup)
+
```

Comment:
I think we don't need these comments now :) can you collapse it back down into the block with no comments like it was before (and the one below as well)